### ISSUE TITLE
installation: issue with install_requires

### ISSUE THREAD

**Issue by jalavik** [inveniosoftware/invenio-demosite#46]
The latest commit b53c743 introduced an error on my local installation:

```
$ pip install -r requirements.txt
Obtaining file:///home/jlavik/envs/pu/src/invenio-demosite (from -r requirements.txt (line 12))
  Running setup.py (path:/home/jlavik/envs/pu/src/invenio-demosite/setup.py) egg_info for package from file:///home/jlavik/envs/pu/src/invenio-demosite
    error in Invenio-Demosite setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers
    Complete output from command python setup.py egg_info:
    error in Invenio-Demosite setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /home/jlavik/envs/pu/src/invenio-demosite
Storing debug log for failure in /home/jlavik/.pip/pip.log
```

Is the `~` valid?

**Comment by jirikuncar** [comment]
Use only `pip install -e .` or upgrade you pip.

**Comment by jalavik** [comment]
Hmm. Did not seem to help:

```
$ pip install -e .
Obtaining file:///home/jlavik/envs/pu/src/invenio-demosite
  Running setup.py (path:/home/jlavik/envs/pu/src/invenio-demosite/setup.py) egg_info for package from file:///home/jlavik/envs/pu/src/invenio-demosite
    error in Invenio-Demosite setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers
    Complete output from command python setup.py egg_info:
    error in Invenio-Demosite setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /home/jlavik/envs/pu/src/invenio-demosite
Storing debug log for failure in /home/jlavik/.pip/pip.log

$ pip install pip --upgrade
Downloading/unpacking pip from https://pypi.python.org/packages/py2.py3/p/pip/pip-6.0.7-py2.py3-none-any.whl#md5=9a7ea5d89062613887b75e01e5d82c36
  Downloading pip-6.0.7-py2.py3-none-any.whl (1.3MB): 1.3MB downloaded
Installing collected packages: pip
  Found existing installation: pip 1.5.6
    Uninstalling pip:
      Successfully uninstalled pip
Successfully installed pip
Cleaning up...

$ hash -r

$ pip install -e .
Obtaining file:///home/jlavik/envs/pu/src/invenio-demosite
    error in Invenio-Demosite setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers
    Complete output from command python setup.py egg_info:
    error in Invenio-Demosite setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers
    
    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /home/jlavik/envs/pu/src/invenio-demosite

$ which pip
/home/jlavik/envs/pu/bin/pip
```

**Comment by jirikuncar** [comment]
Have you upgraded the invenio package first?

**Comment by jalavik** [comment]
Should be latest pu yeah. Still same problems.

Actually, with the latest pip package, there is trouble with reportlab:
```
Collecting pillow==2.0.0,>=2.4.0 (from reportlab<3.2,>=2.7->Invenio==1.9999.5.dev20150128145632)
  Could not find a version that satisfies the requirement pillow==2.0.0,>=2.4.0 (from reportlab<3.2,>=2.7->Invenio==1.9999.5.dev20150128145632) (from versions: 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.7.5, 1.7.6, 1.7.7, 1.7.8, 2.0.0, 2.1.0, 2.2.0, 2.2.1, 2.2.2, 2.3.0, 2.3.1, 2.3.2, 2.4.0, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.6.0, 2.6.1, 2.6.2, 2.7.0)
  No distributions matching the version for pillow==2.0.0,>=2.4.0 (from reportlab<3.2,>=2.7->Invenio==1.9999.5.dev20150128145632)
```
Downgrading pip to 1.5.6 gets around that issue.

Ohh what a dance :dancer:

**Comment by egabancho** [comment]
I have the same issue with `~`

**Comment by jirikuncar** [comment]
Fixed in 5827bbc.

**Comment by drops123456** [comment]
I have the same issue with ~
